### DISCUSSION TITLE
Preload CSS Modules

### DISCUSSION THREAD

**Discussion by arcticmatt** [vercel/next.js#23722] [Help]
### Describe the feature you'd like to request

When I run Lighthouse on my website, it suggests preloading my CSS modules. I don't think there's currently a way to do this, but it seems like it would improve performance.

### Describe the solution you'd like

It would be nice if Next.js did this by default, or provided a setting somewhere to control it.

### Describe alternatives you've considered

I haven't considered any alternatives. 

![image](https://user-images.githubusercontent.com/4356178/113490466-ef820d80-947e-11eb-834a-9159de869002.png)

**Comment by toanbku** [comment]
do we have any update?

**Comment by chungnq253** [comment]
Up, I have the same issue.
